src/ResourceManager/Intune/Commands.Intune.Test/App.config

```diff
@@ -0,0 +1,19 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@rnikhilkumar Remove this filke - it could potentially mask issues in your cmdlets